src/PowerShellEditorServices/Services/DebugAdapter/Debugging/BreakpointDetails.cs

```diff
@@ -55,7 +57,8 @@ private BreakpointDetails()
             int line,
             int? column = null,
             string condition = null,
-            string hitCondition = null)
+            string hitCondition = null,
+            string logMessage = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/app/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4598462)